Source/Scene/Model/CustomShaderPipelineStage.js

```diff
@@ -603,27 +603,37 @@ function addFragmentLinesToShader(shaderBuilder, fragmentLines) {
   shaderBuilder.addFunctionLines(functionId, initializationLines);
 }
 
+const scratchShaderLines = [];
 function addLinesToShader(shaderBuilder, customShader, generatedCode) {
   const vertexLines = generatedCode.vertexLines;
+
+  const shaderLines = scratchShaderLines;
```

Comment:
nitpick about line breaks:

```suggestion
const scratchShaderLines = [];

function addLinesToShader(shaderBuilder, customShader, generatedCode) {
  const vertexLines = generatedCode.vertexLines;
  const shaderLines = scratchShaderLines;
```